whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/RdapElasticServiceTestIntegration.java

```diff
@@ -875,8 +875,8 @@ public void search_redactions() {
                 .map(RdapObject::getRedacted)
                 .flatMap(Collection::stream)
                 .map(Redaction::getPrePath)
-                .collect(Collectors.toList()), containsInAnyOrder("$.vcardArray[1][?(@[0]=='notify')]", "$.vcardArray[1][?" +
-                "(@[0]=='e-mail')]", "$.entities[?(@.handle=='PP1-TEST')].vcardArray[1][?(@[0]=='e-mail')]"));
+                .collect(Collectors.toList()), containsInAnyOrder("$.vcardArray[1][?(@[0]=='notify')]",
```

Comment:
Remove notify as redacted (but not e-mail)

Reply:
Done